spec.md

```diff
@@ -290,3 +294,43 @@ The following example shows a CloudEvent serialized as JSON:
     "data" : "<much wow=\"xml\"/>"
 }
 ```
+
+Regarding the resolution algorithm, lets consider:
```

Comment:
This section confuses me. It sounds a bit like text that should be in the main part of the spec, not in an example section, because its trying to explain how to resolve relative URIs - which feels like normative language.  But, it's also just reiterating what RFC3986 says - so why do we need to repeat it?

Also, where is "base URI" coming from?  We don't have that defined as part of the spec. 

Perhaps it would help me if you could explain what you're trying to fix, or clarify, in the spec with this PR.

Reply:
The PR is based on the issue [332](https://github.com/cloudevents/spec/issues/332) opened by. The [example](https://github.com/cloudevents/spec/blob/master/spec.md#example) of serialized JSON is not valid against the [JSON Schema](https://github.com/cloudevents/spec/blob/master/spec.json), because of `source` element value, which is `/mycontext`.

Reply:
Yup - but I think an easier solution is to just change the example to use a real URI or to make the type of "source" into a URI-reference.  Getting into URI resolution seems unnecessary to me.

Reply:
I totally agree with you @duglin!

I will refactor the PR!